pom.xml

```diff
@@ -40,24 +40,28 @@
   </developers>
 
   <scm>
-    <connection>scm:git:git@github.com:sagacify/sonar-scala.git</connection>
-    <developerConnection>scm:git:git@github.com:sagacify/sonar-scala.git</developerConnection>
-    <url>https://github.com/sagacify/sonar-scala</url>
+    <connection>scm:git:git@github.com:sbartram/sonar-scala.git</connection>
+    <developerConnection>scm:git:git@github.com:sbartram/sonar-scala.git</developerConnection>
+    <url>https://github.com/sbartram/sonar-scala</url>
     <tag>HEAD</tag>
   </scm>
 
   <issueManagement>
-    <url>https://github.com/sagacify/sonar-scala/issues</url>
+    <url>https://github.com/sbartram/sonar-scala/issues</url>
   </issueManagement>
 
   <properties>
-    <sonar.version>5.4</sonar.version>
+    <license.title>SonarQube Scala</license.title>
+    <license.years>2009-2018</license.years>
+    <license.owner>SonarSource SA</license.owner>
+    <license.mailto>mailto:info AT sonarsource DOT com</license.mailto>
+    <sonar.version>6.5</sonar.version>
     <sonar.pluginKey>scala</sonar.pluginKey>
     <sonar.pluginName>Scala</sonar.pluginName>
     <sonar.pluginClass>com.sagacify.sonar.scala.ScalaPlugin</sonar.pluginClass>
 
-    <scala.version>2.11.8</scala.version>
-    <scala.major.version>2.11</scala.major.version>
+    <scala.version>2.10.7</scala.version>
```

Comment:
@pcross616, any particular reason for dropping the minor version to 2.10? The 2.10.x series is no longer actively maintained.

Reply:
Might have been a miss take. let me check i honestly dont remember,